src/reader_util.cpp

```diff
@@ -374,3 +376,20 @@ std::string ReaderUtil::Recode(const std::string& str_to_encode,
 	return result;
 #endif
 }
+
+std::string ReaderUtil::Normalize(const std::string &str) {
+#ifdef LCF_SUPPORT_ICU
+	icu::UnicodeString& uni = icu::UnicodeString(str.c_str()).toLower();
+	UErrorCode err = U_ZERO_ERROR;
+	const icu::Normalizer2* norm = icu::Normalizer2::getNFKCInstance(err);
+	icu::UnicodeString f = norm->normalize(uni, err);
+	std::string res;
+	f.toUTF8String(res);
+	return res;
+#else
+	// LAME
```

Comment:
please don't copy-paste sloppy stuff from my proposed patch, like this // LAME comment.

Anybody having documentation for iconv around? I assume iconv should also support this.

Reply:
https://stackoverflow.com/a/23159081

Reply:
lol, okay nevermind, good that we have ICU.